internal/version/version.go

```diff
@@ -5,20 +5,96 @@ package version
 
 import (
 	"encoding/json"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
 )
 
-// V contains all necessary values for comparing different binary version with each other.
-// It's goal is to limit the startup up outdated binaries as well as having an easier way to ensure the software
-// is on the latest version from a UI/UX perspective
+var (
+	// Timestamp is the UTC timestamp of the compilation time
+	Timestamp string
+	// CommitHash is the git hash of the code being compiled
+	CommitHash string
+	// Version is the semantic version set at compilation
+	// if not a valid semantic version Release should be false
+	Version = "v0.0.1"
+	// Release indicates whether the binary compiled is a release candidate
+	Release bool
+	// Build is a struct containing all relevant build information associated with the binary
+	Build V
```

Comment:
wait if we have this then we shouldn't have the others. we'll just need to update the linker flag. no reason to have both spots

Reply:
oh actually, i get it, V is parsed. let's have some sort of method that returns the Build from the go-release set values instead of a global

Reply:
oh okay okay, init(). okay carry on, ignore me.

Reply:
I am not 100% why we have that. Maybe @coyle has more explanation on that

Reply:
new thought to make it an API with less repetition: do the linker commands work with private values?  maybe we can make the globals release/timestamp/commithash, etc all unexported, and then init uses them to fill in the exported Build value?

Reply:
all the examples I've seen show Global/exported but worth a shot